cmd/rook/ceph/admission.go

```diff
@@ -36,11 +35,9 @@ func init() {
 
 func startAdmissionController(cmd *cobra.Command, args []string) {
 	rook.SetLogLevel()
-
 	rook.LogStartupInfo(admissionCmd.Flags())
-
-	context := rook.NewContext()
-
-	a := admission.New(context, "ceph", operator.ValidateCephResource)
-	a.StartServer()
+	err := operator.StartServer()
```

Comment:
Since we don't have an admission controller struct anymore, let's rename the method. 
```suggestion
	err := operator.StartAdmissionController()
```

Reply:
Done